.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
This rule breaks the code style. Catberry files shouldn't have empty lines at the end.